scripts/skills/actives/rf_swordmaster_charge_skill.nut

```diff
@@ -50,7 +50,7 @@ this.rf_swordmaster_charge_skill <- ::inherit("scripts/skills/actives/rf_swordma
 			id = 11,
 			type = "text",
 			icon = "ui/icons/special.png",
-			text = "Will move to the targeted tile and attack a random adjacent enemy with " + ::MSU.Text.colorPositive((this.m.DamageMultBonus * 100) + "%") + " more damage"
+			text = "Will move to the targeted tile and attack a random adjacent enemy with " + ::MSU.Text.colorPositive((this.m.DamageMultBonus * 100) + "%") + " increased damage"
```

Comment:
We've shifted away from `increased/decreased` and are going with `more/less` for multiplicative modifiers. Is there a particular reason why you prefer `increased` here? If it is because of better language flow then perhaps change `... with x% increased damage` to `... dealing x% more damage`.

Reply:
Yeah better language flow and feel it's more refined language. What do you think?

Reply:
I think in this case `dealing x% more damage` would be the best.